wemake_python_styleguide/visitors/ast/complexity/counts.py

```diff
@@ -72,6 +77,31 @@ def _check_decorators_count(self, node: ModuleMembers) -> None:
                 ),
             )
 
+    def _check_method_decorators(self, node: ModuleMembers) -> None:
+        parent_is_class = isinstance(get_parent(node), ast.ClassDef)
+
+        # Check whether filtered decorators are method decorators
+        is_method_decorator = []
+        for decorator in node.decorator_list:
+            if isinstance(decorator, ast.Name):
+                is_method_decorator.append(
+                    decorator.id in {'classmethod', 'staticmethod', 'property'},
```

Comment:
`{'classmethod', 'staticmethod', 'property'}` needs to be moved into a constant on a class level:

```python
_descriptor_decorators = frozenset((
    'classmethod', 
    'staticmethod', 
    'property',
))
```